R/RunDiagnostics.R

```diff
@@ -675,6 +675,31 @@ executeDiagnostics <- function(cohortDefinitionSet,
     )
   }
 
+  
+  # Defines variables and checcs version of external concept counts table ----------------------
+  if (useExternalConceptCountsTable == FALSE) {
+    conceptCountsTableIsTemp <- TRUE
+  } else {
+    conceptCountsTableIsTemp <- FALSE
+    conceptCountsTable <- "concept_counts"
```

Comment:
why we need to set the conceptCountsTable? In case they have set a temporary table?

Reply:
Yes. We have to define conceptCountsTable, because otherwise, it assigns the name "temp.cohort_counts", and when using the externalConceptCounts, it cannot find the table in the working schema.

Reply:
Ok, but what if they have set conceptCountsTable as argument of executeDiagnostics to something like "my_concept_count", i.e. a custom name. Then you are changing it to the standard name which is not what they expect. I think we should just make sure that it is not a temp table at this point. What do you think?

Reply:
Hey! You are right. In the latest commit I put in place checks to detect if it is a temp table name or not. An error is displayed if the user do not provide the correct external concept counts table. What do you think?